AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -56,7 +62,7 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
         });
     }
 
-    static mapJudgeToModel(judge: JudgeResponse): ParticipantModel {
+    static mapJudgeToModel(judge: JudgeResponse, judgePhone: string): ParticipantModel {
```

Comment:
This method is a bit redundant, you can just use an object init instead. I know this isn't your change, but worth pointing out.

Reply:
Yeah, I'd rewrite this to just do this:

```ts
return {
...this.judge,
title: 'Judge',
is_judge: true,
phone: this.judgePhone,
case_role_name: 'Judge',
hearing_role_name: 'Judge',
username: judge.email	     
}
```

The constructor doesn't do anything and most of the assignments don't.

Reply:
that caused issues later in the method, can we park it for now, and discuss after the merge?